weed/topology/data_node.go

```diff
@@ -166,13 +167,60 @@ func (dn *DataNode) ToMap() interface{} {
 	ret := make(map[string]interface{})
 	ret["Url"] = dn.Url()
 	ret["Volumes"] = dn.GetVolumeCount()
+	ret["VolumesID"] = dn.GetVolumesID()
 	ret["EcShards"] = dn.GetEcShardCount()
 	ret["Max"] = dn.GetMaxVolumeCount()
 	ret["Free"] = dn.FreeSpace()
 	ret["PublicUrl"] = dn.PublicUrl
 	return ret
 }
 
+func (dn *DataNode) GetVolumesID() string {
```

Comment:
There could be a couple of thousands of volume ids on one volume server. This would not "scale".

Reply:
I just limit volumesID display only for max 100 volumes for convenience in the master ui.

Reply:
Thanks! Also, could you please name it as volumeIds instead of volumesId?

Reply:
u r welcome.